src/org/jitsi/impl/neomedia/rtcp/RTCPReceiverFeedbackTermination.java

```diff
@@ -418,7 +418,7 @@ private RawPacket doTransform(RawPacket pkt, boolean send)
                     continue;
                 }
 
-                if (!send)
+                if (!send && pt > -1)
```

Comment:
If I'm reading this patch correctly, is this the only functional difference in this patch?

The PT field should be 7 bits + 1 bit marker and should be accommodated by the 8-bits of the byte with the caveat that any widening operation will have sign extension if we are not careful?

Reply:
Oh now I read the conversation on this change and realize that PT is overloaded with error states in some code paths.  :\

Reply:
The issue was that `PT_PS` & `PT_TL` were in bytes but the `pt` was an `int`. When `PT_PS` was converted to `int` it had a value of -50 and the code never went into the if block.